README.md

```diff
@@ -160,4 +172,52 @@ and clicking `Notify` will trigger a POST request to `https://matterhook.example
     "text": "New code was pushed."
   }
 }
-```
\ No newline at end of file
+```
+
+### Message with menu buttons (supported in Mattermost 5.4+)
+
+You also can post messages with menu buttons. It will post a message with a dropdown button where the users can select a value, which will be posted to the target integration.
+
+Just add an attachment to your message as follows:
+
+```csharp
+Attachments = new List<MattermostAttachment>
+{
+    new MattermostAttachment
+    {
+        Pretext = "This is optional pretext that shows above the attachment.",
+        Text = "This is the text of the attachment. ",
+        Actions = new List<IMattermostAction>
+        {
+            new MattermostMessageMenu
+            {
+                Integration = new MattermostIntegration(config.outgoingWebHookUrl,
+                    new Dictionary<string, object>
+                    {
+                        {"text", "Some data to send always."}
+                    }),
+                Name = "Test",
+                Options = new List<MessageMenuOption>
+                {
+                    new MessageMenuOption("Option1", "value1"),
+                    new MessageMenuOption("Option2", "value2"),
+                    new MessageMenuOption("Option3", "value3")
+                }
+            }
+        }
+    }
+}
+```
+
+You can add as many attachments of any type to a message as you want. Message menus and buttons can be used to for ex.:
+
+* Mark a task complete in your project management tracker
+* Conduct a customer survey or a poll
+* Initiate a command to merge a branch into a release
+
+## Contributing
+
+We welcome everyone who wants to contribute to this repo!
+For it, just open an issue with your intention or make a comment in some open issue you would like to work on. 
```

Comment:
Line 221 - Change to read:

`Just open an issue......` etc. No need for the `For it,` part :)

Maybe also add the following line underneath:

> Please ensure that your Pull Requests are based on, and submitted against the `development` branch.

Reply:
Done.